payloads/v9/channel.ts

```diff
@@ -1877,6 +1925,77 @@ export interface APITextInputComponent extends APIBaseComponent<ComponentType.Te
 	required?: boolean;
 }
 
+export enum UnfurledMediaItemLoadingState {
+	Unknown,
+	Loading,
+	LoadedSuccess,
+	LoadedNotFound,
+}
+
+export interface APIUnfurledMediaItem {
+	/**
+	 * Supports arbitrary urls _and_ attachment://<filename> references
+	 */
+	url: string;
+	proxy_url?: string;
+	width?: number | null;
+	height?: number | null;
+	placeholder?: string | null;
+	placeholder_version?: number | null;
+	content_type?: string | null;
+	loading_state?: UnfurledMediaItemLoadingState;
+	flags?: number;
```

Comment:
Unclear if discord-api-types has a separation between request and response models, but `url` is the only dev-settable field. All others are provided by discord.